src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java

```diff
@@ -224,6 +224,39 @@ public ResponseEntity<ProgrammingExerciseStudentParticipation> resumeParticipati
         return ResponseEntity.ok().body(participation);
     }
 
+    /**
+     * PUT exercises/:exerciseId/request-feedback: Requests manual feedback for the latest participation
+     *
+     * @param exerciseId of the exercise for which to resume participation
+     * @param principal  current user principal
+     * @return ResponseEntity with status 200 (OK)
+     */
+    @PutMapping("exercises/{exerciseId}/request-feedback")
+    @PreAuthorize("hasRole('USER')")
+    @FeatureToggle(Feature.ProgrammingExercises)
+    public ResponseEntity<ProgrammingExerciseStudentParticipation> requestFeedback(@PathVariable Long exerciseId, Principal principal) {
```

Comment:
**CR**: The PR description states that this should only be possible if 100% of points are reached from the automatic tests. Where is this ensured?

Reply:
check added https://github.com/ls1intum/Artemis/pull/5670/commits/94e118d5b01f37ee716ce54ff7ec9fc13849d6fc